bundles/org.openhab.binding.energidataservice/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="energidataservice" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>Energi Data Service Binding</name>
+	<description>This is the binding for Energi Data Service.</description>
```

Comment:
Could you be a bit more verbose, so that somebody who hasn't heard about it gets a clue what this is about?

Reply:
Good idea, how does it look to you now? The service provides so many different datasets, so it's hard to be very concrete, but at least it should be mentioned it's about energy data.